ref/README.md

```diff
@@ -1,78 +1,160 @@
+# The C_SW reference kernel
+
+NOTE: If you are reading this with a plain text editor, please note that this document is
+formatted with Markdown syntax elements.  See https://www.markdownguide.org/cheat-sheet/
```

Comment:
This looks a little cleaner to me : 
NOTE: If you are reading this with a plain text editor, please note that this document is
formatted with [ Markdown syntax elements](https://www.markdownguide.org/cheat-sheet/).

Reply:
I purposely did not use Markdown syntax in that note in order to not confuse people.  But, yes, normally you would use the [name](link) syntax to do that.